<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Searching</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="2">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>Consists of letters</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Find &amp;next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>&amp;Save all to file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Contains the las</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>&amp;Contains letters</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
